analyzers/its/expected/akka.net/Akka.DistributedData--netstandard2.0-S2326.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S2326",
+"message":  "'T' is not used in the interface.",
+"location":  {
+"uri":  "sources\akka.net\src\contrib\cluster\Akka.DistributedData\Key.cs",
```

Comment:
That's a false positive:
```cs
public interface IKey<out T> : IKey where T : IReplicatedData { } // Compliant. Used in a type constraint
```

Reply:
As discussed offline, we'll declare it as TP